account_invoice_transmit_method/models/partner.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017-2020 Akretion France (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    customer_invoice_transmit_method_id = fields.Many2one(
+        "transmit.method",
+        string="Customer Invoice Transmission Method",
+        company_dependant=True,
```

Comment:
this is wrong (in v13 as well)
```suggestion
        company_dependent=True,
```
please check https://github.com/OCA/account-invoicing/pull/822 and maybe help that one get merged :pray: